pennylane/ops/op_math/__init__.py

```diff
@@ -68,8 +68,8 @@
 from .pow import Pow, pow
 from .prod import Prod, prod
 from .sprod import SProd, s_prod
-from .sum import Sum, sum, op_sum
-from .symbolicop import SymbolicOp
+from .sum import Sum, op_sum, sum
+from .symbolicop import ScalarSymbolicOp, SymbolicOp
```

Comment:
I can't comment there but can you add `~ScalarSymbolicOp` to the "Symbolic Classes" section above? Line 52

Reply:
Added!